app/uk/gov/hmrc/nisp/views/main.scala.html

```diff
@@ -55,13 +58,16 @@
   }
 }
 
-@scriptElement = @{
+@scriptElement = {
+  @if(ApplicationConfig.isGtmEnabled) {
+    @views.html.google_tag_manager(ApplicationConfig.googleTagManagerId)
+  }
     Seq(
```

Comment:
Alignment

Reply:
Does this work? Not sure how twirl will render this but scala would only return the last thing on the page which is the `Seq[Map[_,_]]`

Reply:
It does seem to be getting different values from the Seq